src/main/java/org/terasology/smithing/event/OpenCharcoalPitRequest.java

```diff
@@ -17,5 +17,8 @@
 
 import org.terasology.network.NetworkEvent;
 
+/*
+* Sends request over network to activate the charcoal pit
```

Comment:
The event itself doesn't really send the request, it is more like it *is* the request being sent. So you could say something like "Network event used to activate the charcoal pit"